staging/src/k8s.io/cloud-provider/controllers/node/config/v1alpha1/defaults.go

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+func RecommendedDefaultNodeControllerConfiguration(obj *NodeControllerConfiguration) {
+	if obj.WorkerCount <= 0 {
```

Comment:
don't change any value other than 0... a negative value is a configuration error we should reject

Reply:
Done